internal/pointer/pointer.go

```diff
@@ -21,6 +21,14 @@ import (
 	"golang.org/x/exp/constraints"
 )
 
+func GetOrZero[T any](ptr *T) T {
```

Comment:
empty? seems more go idiomatic for what it does

Reply:
The reason on why I went with Zero is because that is the go terminology (example in https://go.dev/tour/basics/12)